panel/pane/echarts.py

```diff
@@ -38,6 +38,10 @@ class ECharts(ModelPane):
     object = param.Parameter(default=None, doc="""
         The Echarts object being wrapped. Can be an Echarts dictionary or a pyecharts chart""")
 
+    options_kwargs = param.Parameter(default=None, doc="""
+        An optional dict of options passed to Echarts.setOption. Allows to fine-tune the rendering behavior.
```

Comment:
It might be useful to add "For example if you want to update your chart to a chart containing a smaller number of series setting `options_kwargs={ "replaceMerge": ['series'] })` is needed".